seq2science/scripts/scRNA-seq-1.0.0-rc.1/data/MT_genes.txt

```diff
@@ -0,0 +1,37 @@
+ND1
```

Comment:
Is there an option in the config to use a custom geneset? Fine to keep this list as default!

Reply:
Removed this one since it can be set via config.yaml.